backend/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "es6",
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "outDir": "./prod/"
+  },
+  "files": [
+    "./dev/app.ts",
+    "./dev/controller.ts",
+    "./dev/model.ts",
+    "./dev/router.ts",
+  ]
```

Comment:
if we add any new file will it need to be added to this list? can you try to make it more versatile? (for example, let's process ALL files except ...)

Reply:
Now I`m using:
` "include": [
    "./dev/*.ts"
  ]`